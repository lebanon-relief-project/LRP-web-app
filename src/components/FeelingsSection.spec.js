import React from "react";
import { render } from "@testing-library/react";
import FeelinsSection from "./FeelingsSection";

describe("the OurMission component", () => {
  let container;
  it("should match the snapshot", () => {
    ({ container } = renderFeelingsSection());

    expect(container).toMatchSnapshot();
  });
});

const renderFeelingsSection = () => {
  return render(<FeelinsSection />);
};
