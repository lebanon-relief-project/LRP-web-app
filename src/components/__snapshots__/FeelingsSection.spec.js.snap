// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the OurMission component should match the snapshot 1`] = `
<div>
  <section
    class="sc-gtssRu jGobNw"
  >
    <div
      class="sc-bdnylx icjZAh"
    >
      <h2
        class="sc-dlnjPT hxnyXJ"
      >
        Learn more and select the ones that you have experienced
      </h2>
      <p
        class="sc-hKFyIo jEDpVH"
      >
        Below are flashcards with the most common presentations of traumatic stress. Click on those that you have experienced to learn more about your reaction and ways to manage.
      </p>
    </div>
  </section>
</div>
`;
